backend/src/main/java/gov/cdc/usds/simplereport/service/LoggedInAuthorizationService.java

```diff
@@ -60,6 +60,8 @@ public List<OrganizationRoleClaims> findAllOrganizationRoles() {
       String username = currentAuth.getName();
       List<OrganizationRoleClaims> dbOrgRoleClaims =
           _dbOrgRoleClaimsService.getOrganizationRoleClaims(username);
+      _dbOrgRoleClaimsService.checkOrgRoleClaimsEquality(
```

Comment:
⚠️ Anyone know how often we reset `organizationRolesContext` and have to fetch it again? 🤔 

Based on [digging around in Azure](https://portal.azure.com/#@cdc.onmicrosoft.com/resource/subscriptions/7d1e3999-6577-4cd5-b296-f518e5c8e677/resourceGroups/prime-simple-report-dev/providers/Microsoft.Insights/components/prime-simple-report-dev2-insights/searchV1), it seems like on initial log in and page load `fetchCurrentOrganizationRoles` (which is the only method that calls `findAllOrganizationRoles` is called a bunch of times and then not again for 20-30 mins or so... 

I'm asking because I'm wondering if we are doing the check too frequently if I set the check at this point. 🤔

Reply:
Don't know off the top of my head.  And not sure if the better place to look would be spring docs or okta's 🤔

Reply:
does this line mean that the context is only valid for a single request? if so it seems like it would need to be refetched any time we are handling a new request that needs the current org roles
https://github.com/CDCgov/prime-simplereport/blob/f800b082b3ee8349e2747faef417b095f8613bf2/backend/src/main/java/gov/cdc/usds/simplereport/api/CurrentOrganizationRolesContextHolder.java#L11

Reply:
fwiw since we are condensing the logs into a single alert per day (?) it doesn't seem that harmful if we are logging frequently. though it may be a little bit of a pain to parse through the log lines later I don't see a super simple alternative 🤷

Reply:
Thank you for digging into that @mehansen. Based on what you shared, I'm surprised I didn't see it log more frequently when testing. 🤷 After I merge, I'll keep on eye on prod and the logs...